### PR TITLE
Fixed tabs header for cool usability

### PR DESCRIPTION
**Pull request by awinogradov**

### CODE REVIEW

File: src/blocks/common.blocks/block/block.styl [hunk 1]
```diff
@@ -3,11 +3,13 @@
     .tabs__header,
     .tabs__content
```
Comment (tadatuta) [inline src/blocks/common.blocks/block/block.styl]: i'd remove `.tabs__content` from here, revert paddings for `tabs__header` to `0 30px` and add `padding: 40px 30px 20px` for `tabs__content` as separate selector

// cc @tavriaforever

Comment (awinogradov) [inline src/blocks/common.blocks/block/block.styl]: @tadatuta i'd move tabs styles to `tabs` block

File: src/blocks/common.blocks/tabs/_theme/tabs_theme_default.styl [hunk 2]
```diff
@@ -3,6 +3,14 @@
     /* Верхняя черта */
     &__header
```
Comment (tavriaforever) [inline src/blocks/common.blocks/tabs/_theme/tabs_theme_default.styl:4]: Some elements, has `position:relative` & their child has `z-index`, so they overlap `tabs__header` https://yadi.sk/i/2uhQ8qsMbmnvf

You can add z-index more than `1`. I would write `100`

**Comment by tavriaforever** [comment]
Good improvement, I like it

File: src/blocks/common.blocks/block/block.styl (hunk 1 above)
Comment (tavriaforever) [inline src/blocks/common.blocks/block/block.styl]: @tadatuta :+1:

File: src/blocks/common.blocks/tabs/_theme/tabs_theme_default.styl (hunk 2 above)
Comment (awinogradov) [inline src/blocks/common.blocks/tabs/_theme/tabs_theme_default.styl:4]: @tavriaforever arrow is cool :+1:

**Comment by tavriaforever** [comment]
Thanks!

File: src/blocks/common.blocks/tabs/_theme/tabs_theme_default.styl (hunk 2 above)
Comment (tavriaforever) [inline src/blocks/common.blocks/tabs/_theme/tabs_theme_default.styl:4]: :smile:

**Pull request merged**